selectDialog_view.py

```diff
@@ -7,6 +7,10 @@
 
 
 class SelectDialog(QDialog):
+    """Widget for displaying data (represented as list) in table form
```

Comment:
What form? select table form

Reply:
Same as previous comment, what form of representation do you want to see, table form means that data is represented as table, it's kind of an abstraction.